### PR TITLE
delete "Simple_Robot_Motor_Control"

### PR DESCRIPTION
**Pull request by Mirco04**
delete https://github.com/Mirco04/Simple_Robot_Motor_Control

### CODE REVIEW

**Review by per1234 — APPROVED** [review]

**Comment by per1234** [comment]
Hello @Mirco04. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests and will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Comment by umbynos** [comment]
Done in https://github.com/arduino/library-registry/pull/1526#issuecomment-1163286555

**Pull request merged**
